engines/sherlock/animation.cpp

```diff
@@ -0,0 +1,208 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ */
+
+#include "sherlock/animation.h"
+#include "sherlock/sherlock.h"
+#include "common/algorithm.h"
+
+namespace Sherlock {
+
+static const int NO_FRAMES = FRAMES_END;
+
+Animation::Animation(SherlockEngine *vm): _vm(vm) {
+}
+
+/**
+ * Play a full-screen animation
+ */
+bool Animation::play(const Common::String &filename, int minDelay, int fade,
+		bool setPalette, int speed) {
+	Events &events = *_vm->_events;
+	Screen &screen = *_vm->_screen;
+	Sound &sound = *_vm->_sound;
+	int soundNumber = 0;
+	sound._playingEpilogue = true;
```

Comment:
Is this field ever read? I cannot seem to find any read access through GitHub's view.

The name itself also seems a bit confusing, because this seems to be always set, even when there's a "_titleOverride" set and the method is called "play" in a generic fashion. So, this seems more like "_playingAnimation" or something to me. But since I can't find any use it's hard to say what it is about...